content/developer/addons/addon-visibility.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Addon Visibility
+tags:
+- Theming
+- Plugin
+- Addon
+- Visibility
+category: addons
+menu:
+  developer:
+    weight: 11
+    parent: addons
+aliases:
+- /theming/visibility
```

Comment:
Was that a real link? Is this a remnant from an old file?

Reply:
It seems you added it when the file was creating. It's probably not necessary.

https://github.com/vanilla/docs/blame/2e283f7466eda728db98ebe6c2a0da4c30915d73/content/developer/theming/themevisibility.md#L11